AnEvilPenguin.Calculator/CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Newtonsoft.Json;
+using System.Diagnostics;
```

Comment:
Please remove unused using. 

![image](https://user-images.githubusercontent.com/108711476/215914343-6c6f197a-0986-484d-9e60-050705604cbc.png)

Reply:
Done! Annoyed I missed that!